packages/example-app-base/src/components/FeedbackList/FeedbackList.styles.ts

```diff
@@ -0,0 +1,26 @@
+import { getTheme, mergeStyleSets, DefaultFontStyles, getFocusStyle } from 'office-ui-fabric-react/lib/Styling';
+
+const theme = getTheme();
+export const classNames = mergeStyleSets({
```

Comment:
So far I'm converting *only* the former `ms-ListGhostingExample-*` classes to css-in-js. The rest of the conversions will come later.

Reply:
Hmm, another option would be to replace the divs with Stacks and just use the `styles` prop...

Reply:
Not quite sure offhand what that would look like? I'd prefer to leave this as-is for this PR and possibly come back to it later.